libp2p/varint.nim

```diff
@@ -72,6 +73,7 @@ proc getUVarint*[T: PB|LP](vtype: typedesc[T],
   when vtype is PB:
     const MaxBits = byte(sizeof(outval) * 8)
   else:
+    var zcounter = 0
```

Comment:
a less intrusive way is to calculate the size of the result using something like https://github.com/status-im/nim-stew/blob/f5be0ab63a64c314e74223891a7651e2eaaa8fec/stew/varints.nim#L210 and compare with the number of bytes actually consumed - this seems more simple and avoids branching in each loop iteration

Reply:
done.